src/Nix/Diff.hs

```diff
@@ -98,7 +90,7 @@ data Orientation = Character | Word | Line
     underneath `/nix/store`, but this is the overwhelmingly common use case
 -}
 derivationName :: StorePath -> Text
-derivationName = Text.dropEnd 4 . Text.drop 44 . Text.pack . unsafeStorePathFile
+derivationName = Text.dropEnd 4 . Text.drop 44 . Text.pack . (.unsafeStorePathFile)
```

Comment:
Very minor thing, but if we're switching to record dot notation then I'd be more inclined to not make this point-free:

```suggestion
derivationName path = Text.dropEnd 4 (Text.drop 44 (Text.pack path.unsafeStorePathFile))
```

Similar comment in a few other places